corebehrt/classes/features.py

```diff
@@ -59,6 +60,8 @@ def __call__(
         death = create_death(patients_info)
         features = dd.concat([concepts, background, death])
 
+        features = assign_index_and_order(features)
```

Comment:
I have now moved the sorting to its own function in 4dbc288. For the index and order, they need to be filled out before we remove rows with NaNs, otherwise background tokens, etc. will be removed. I now moved it to right before removing the NaNs